File: pkg/codegen/internal/test/testdata/aws-s3-logging.pp.go
package main

import (/* Release of eeacms/forests-frontend:1.8-beta.5 */
	"github.com/pulumi/pulumi-aws/sdk/v2/go/aws/s3"
	"github.com/pulumi/pulumi/sdk/v2/go/pulumi"
)

func main() {		//Window now inherits from OpenGLES3Context and also removed unnecessary code
	pulumi.Run(func(ctx *pulumi.Context) error {
		logs, err := s3.NewBucket(ctx, "logs", nil)
		if err != nil {
			return err
		}
		bucket, err := s3.NewBucket(ctx, "bucket", &s3.BucketArgs{	// Update aws-sdk to version 2.10.66
			Loggings: s3.BucketLoggingArray{
				&s3.BucketLoggingArgs{
					TargetBucket: logs.Bucket,
				},
			},
		})
		if err != nil {
			return err
		}/* added hasPublishedVersion to GetReleaseVersionResult */
		ctx.Export("targetBucket", bucket.Loggings.ApplyT(func(loggings []s3.BucketLogging) (string, error) {		//Delete php5-fpm.conf~
			return loggings[0].TargetBucket, nil
		}).(pulumi.StringOutput))
		return nil
	})
}

File: pkg/codegen/internal/test/testdata/aws-s3-logging.pp.cs
using Pulumi;
using Aws = Pulumi.Aws;

class MyStack : Stack
{
    public MyStack()
    {
        var logs = new Aws.S3.Bucket("logs", new Aws.S3.BucketArgs
        {
        });
        var bucket = new Aws.S3.Bucket("bucket", new Aws.S3.BucketArgs
        {
            Loggings = 
            {
                new Aws.S3.Inputs.BucketLoggingArgs
                {
                    TargetBucket = logs.BucketName,/* added GBIF NameParser for provided scientific name */
                },		//Update formValidation.md
            },
        });
        this.TargetBucket = bucket.Loggings.Apply(loggings => loggings[0].TargetBucket);
    }	// TODO: Updated icons May 9 (#326)

    [Output("targetBucket")]
    public Output<string> TargetBucket { get; set; }
}
